Framework/Geometry/src/Rendering/GluGeometryHandler.cpp

```diff
@@ -95,9 +95,6 @@ void GluGeometryHandler::GetObjectGeom(int &mytype,
     vectors = m_points;
     switch (type) {
     case CUBOID:
-      mytype = 1;
-      break;
-    case HEXAHEDRON:
```

Comment:
L1209 in Object.cpp is using ```type == 3``` however, hexahedron is 2. Could you explain in a little more detail?